redbot/cogs/mod/mod.py

```diff
@@ -67,7 +68,7 @@ def __init__(self, bot: Red):
 
         self.settings = Config.get_conf(self, 4961522000, force_registration=True)
         self.settings.register_global(**self.default_global_settings)
-        self.settings.register_guild(**self.default_guild_settings)
+        self.settings.register_guild(**self.default_guild_settings, force_registration=True)
```

Comment:
```suggestion
        self.settings.register_guild(**self.default_guild_settings)
```
this shouldn't be here